docs/runtime_attributes.md

```diff
@@ -0,0 +1,19 @@
+# Key Runtime Attributes
+
+This doc is intended to capture the key runtime attributes for Node.js.  
+These attributes will be used along with the use-cases in order to
+guide what we develop/run as part of the community benchmarking
+efforts.
+
+# Attributes
+
+The following attributes have been identified so far:
+
+* startup/stop time
```

Comment:
i'd say the first interesting metric is when `LoadEnvironment()` runs. That's essentially the bootstrap before node enters the event loop, and when most of the modules are loaded. As for stop, it is zero, plus any `'exit'` callbacks.